app/controllers/admin/routes.py

```diff
@@ -148,7 +147,7 @@ def rsvpLogDisplay(eventId):
     eventData = model_to_dict(event, recurse=False)
     eventData['program'] = event.program
     isProgramManager = g.current_user.isProgramManagerFor(eventData['program'])
-    if g.current_user.isCeltsAdmin or (g.current_user.isCeltsStudentStaff and isProgramManager):
+    if g.current_user.isCeltsAdmin or g.current_user.isCeltsStudentAdmin or (g.current_user.isCeltsStudentStaff and isProgramManager):
```

Comment:
You don't need the `isCeltsStudentStaff` check here, because all we care about is whether they are a program manager (and that will likely mean student staff is redundant)